sdk/loadtestservice/azure-developer-loadtesting/samples/create_or_update_test.py

```diff
@@ -0,0 +1,54 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+FILE: create_or_update_test.py
+
+DESCRIPTION:
+    This sample shows how to create or update a load test
+
+USAGE:
+    python create_or_update_test.py
+
+    Set the environment variables with your own values before running the sample:
+    1)  AZURE_CLIENT_ID - client id
+    2)  AZURE_CLIENT_SECRET - client secret
+    3)  AZURE_TENANT_ID - tenant id for your Azure
+"""
+from azure.developer.loadtesting import LoadTestingClient
+
+# for details refer: https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/loadtestservice/azure-developer-loadtesting/README.md
+from azure.identity import DefaultAzureCredential
+
+import os
+from dotenv import load_dotenv
+
+load_dotenv()
+LOADTESTSERVICE_ENDPOINT = os.environ["LOADTESTSERVICE_ENDPOINT"]
+
+# Build a client through AAD and resource endpoint
+client = LoadTestingClient(credential=DefaultAzureCredential(), endpoint=LOADTESTSERVICE_ENDPOINT)
+
+# ID to be assigned to test
+TEST_ID = "my-new-sdk-test-id"
+
+result = client.load_test_administration.create_or_update_test(
+    TEST_ID,
+    {
+        "description": "",
+        "displayName": "My New Load Test",
+        "loadTestConfig": {
+            "engineInstances": 1,
+            "splitAllCSVs": False,
+        },
+        "secrets": {},
+        "environmentVariables": {},
+        "passFailCriteria": {"passFailMetrics": {}}
```

Comment:
Can we provide some values for secrets, environment variables and criteria? You can refer to example values here https://learn.microsoft.com/en-us/azure/load-testing/reference-test-config-yaml

Reply:
fixed at: https://github.com/Azure/azure-sdk-for-python/pull/25550/commits/3de6e15979916a53b9bacc4499bb7eb4452b5cde